lib/seahorn/BvOpSem2.cc

```diff
@@ -2716,9 +2750,10 @@ Expr Bv2OpSemContext::getRawMem(Expr p) { return mem().getRawMem(p); }
 
 Bv2OpSem::Bv2OpSem(ExprFactory &efac, Pass &pass, const DataLayout &dl,
                    TrackLevel trackLvl)
-    : OperationalSemantics(efac), m_pass(pass), m_trackLvl(trackLvl),
-      m_td(&dl) {
+    : OperationalSemantics(efac), m_pass(pass), m_trackLvl(trackLvl), m_td(&dl),
+      m_cfg_builder_man(nullptr), m_crab_rng_solver(nullptr) {
```

Comment:
I don't think these are needed. Unique pointer are null by default. That's the default constructor